lib/elixir/scripts/elixir_docs.exs

```diff
@@ -28,6 +31,7 @@ canonical = System.fetch_env!("CANONICAL")
   ],
   groups_for_extras: [
     "Anti-patterns": ~r"pages/anti-patterns/.*\.md$",
+    "Meta-programming": ~r"pages/meta-programming/.*\.md$",
```

Comment:
I am wondering if this should come after references but anyway works for me. :)

Reply:
I think we can figure out a good order after we move all the guides really 🙃